envs/conda/cubi_tools.yaml

```diff
@@ -10,3 +10,4 @@ dependencies:
   - pylint=2.14.5
   - isort=5.10.1
   - black=22.6.0
+  - requests=2.25.1
```

Comment:
I may have missed it, but why is that dependency necessary?

Reply:
I forgot to delete this dependency after I replaced the part where it was needed. It is irrelevant now and has been removed.